service/src/test/java/bio/terra/profile/service/profile/flight/create/CreateProfileFlightTest.java

```diff
@@ -69,6 +87,11 @@ void createGcpProfileSuccess() {
     assertNotNull(createdProfile.createdTime());
     assertNotNull(createdProfile.lastModified());
     assertEquals(createdProfile.createdBy(), userRequest.getEmail());
+
+    var timer = meterRegistry.find("profile.creation.time").timer();
+    assertNotNull(timer);
+    assertEquals(timer.count(), 1);
+    assertEquals(timer.getId().getTag("cloudPlatform"), CloudPlatform.GCP.toString());
```

Comment:
Chose not to create dedicated tests for profile creation because there is quite a bit of setup/mocking.